pkgs/flox-interpreter/default.nix

```diff
@@ -62,17 +62,19 @@ let
   };
 
 in
-runCommandNoCC "flox-activation-scripts"
+runCommandNoCC "flox-interpreter"
   {
     nativeBuildInputs = [ gnused ];
     outputs = [
       "out"
-      "build_wrapper"
+      "wrapper"
```

Comment:
(non-blocking) I'm unsure about dropping the "build_" prefix from "wrapper" in the names other than the directory structure because I currently find myself double-taking about what it's wrapping and why it's different from plain "out".

Reply:
i removed "build" because its kinda imprecise as well, i.e. it is used _during_ the build and for executables resulting from a build, but `build wrapper` sounds like a greater scope.

Reply:
Yeah, that works 👍🏻